pkg/pillar/hypervisor/xen.go

```diff
@@ -163,6 +163,11 @@ func (ctx xenContext) CreateDomConfig(domainName string,
 	xenGlobal := ""
 	uuidStr := fmt.Sprintf("appuuid=%s ", config.UUIDandVersion.UUID)
 
+	if config.EnforceNetworkInterfaceOrder {
```

Comment:
You are probably right, especially since this flag will be configured across a fleet of devices, some of which could be using a different hypervisor and this change would make them unusable.
I changed it and EVE will now just log error and continue.